client/landing/stepper/declarative-flow/internals/steps-repository/importer/index.tsx

```diff
@@ -213,11 +232,62 @@ export function withImporterWrapper( Importer: ImporterCompType ) {
 					fromSite={ fromSite }
 					urlData={ fromSiteData ?? undefined }
 					stepNavigator={ stepNavigator }
+					renderHeading={ ! useContainerV2 }
 				/>
 			);
 		};
 
 		const importJob = getImportJob( importer );
+
+		if ( useContainerV2 ) {
+			const importerData = getImportDragConfig( importer, stepNavigator?.supportLinkModal );
```

Comment:
Looks like the supportLinkModal is always false, I didn't find any reference where we are setting it to true. 🤔

Reply:
mmm it's used like that allover and it's hardcoded to `false` here. I am inclined to leave it as is just to conform with the old way of doing and if we find out that we should hardcode the value to `true`. Frankly, looks like cruft, but I better be safe that sorry here.